src/common/constants/index.ts

```diff
@@ -0,0 +1 @@
+export const CACHE_TTL = 195 * 60 * 1000; // 195 minutes in milliseconds
```

Comment:
> i'd like to understand the cost impact here to make an informed tradeoff for how frequently we want to update the cache: https://cloud.google.com/products/calculator?hl=en
> 
> if the cost is negligible i'd optimize for more up-to-date data, i.e. shorter TTL (an hour or less)

So the cache TTL does not impact the cost. CACHE_TTL defines how long the cached data is stored in memory. The cost is dictated by a couple of factors. If we're just focused on GCP, the cost depends on whether there are instances of the API running. This can be configured: we can either always have an instance running for the best experience or set the minimum instances to 0. When the minimum instances are set to 0, the API scales down if it’s not receiving traffic, and we are charged per request. For the current configuration (512 MB + 1 vCPU) with a minimum and maximum of 1 instance, the cost is estimated to be around ~$50.

This only accounts for the GCP side of things. If we want to increase the cache warmup frequency (currently set to every 3 hours), we need to think about the credits for all the external APIs we’re using. I intentionally set this to 3 hours, as it results in 8 real requests in a 24-hour period, which is the average number of hits the reserve site was getting. At this rate, I don’t see us using more credits than we would have otherwise. If we make the cache warmup more frequent, we’ll need to look into how that impacts API usage.

For now I'd just recommend we keep things as is and review usage in a month we can then make any changes if needed.

Reply:
I'll update the readme to include more information on the cache in another PR.